clarity/src/vm/tests/traits.rs

```diff
@@ -37,6 +37,7 @@ use crate::vm::ContractContext;
 #[case(ClarityVersion::Clarity1, StacksEpochId::Epoch2_05)]
 #[case(ClarityVersion::Clarity1, StacksEpochId::Epoch21)]
 #[case(ClarityVersion::Clarity2, StacksEpochId::Epoch21)]
+#[case(ClarityVersion::Clarity2, StacksEpochId::Epoch22)]
```

Comment:
Are you going to include epoch 2.3 here as well?

Reply:
Sure, will add.